streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkClusterWatcher.java

```diff
@@ -321,11 +251,9 @@ private FinalApplicationStatus stringConvertFinalApplicationStatus(String value)
    * @return
    */
   private ClusterState finalApplicationStatusConvertClusterState(FinalApplicationStatus status) {
-    switch (status) {
-      case UNDEFINED:
-        return ClusterState.RUNNING;
-      default:
-        return ClusterState.STOPPED;
+    if (status == FinalApplicationStatus.UNDEFINED) {
```

Comment:
When the yarn application was accepted by yarn resourcemanager, the FinalApplicationStatus is `UNDEFINED` too, but the flink cluster isn't `RUNNING`

This case was mentioned in https://github.com/apache/incubator-streampark/pull/2541#discussion_r1162899796

Reply:
> When the yarn application was accepted by yarn resourcemanager, the FinalApplicationStatus is `UNDEFINED` too, but the flink cluster isn't `RUNNING`
> 
> This case was mentioned in [#2541 (comment)](https://github.com/apache/incubator-streampark/pull/2541#discussion_r1162899796)

I also have questions about this logic and further discussion is needed. @xujiangfeng001

Reply:
This is my negligence. The corresponding logic was not modified during the implementation of this piece of content. The logic here does not need to determine the `finalStatus `of the application, only the `application state` needs to be determined as `running` to consider the `flink cluster` as `running`.

```
public enum YarnApplicationState {
  /** Application which was just created. */
  NEW,

  /** Application which is being saved. */
  NEW_SAVING,

  /** Application which has been submitted. */
  SUBMITTED,

  /** Application has been accepted by the scheduler */
  ACCEPTED,

  /** Application which is currently running. */
  RUNNING,

  /** Application which finished successfully. */
  FINISHED,

  /** Application which failed. */
  FAILED,

  /** Application which was terminated by a user or admin. */
  KILLED
}
```